src/services/school/model.js

```diff
@@ -92,6 +92,7 @@ const schoolSchema = new Schema(
 		 */
 		enableStudentTeamCreation: { type: Boolean, required: false },
 		inMaintenanceSince: { type: Date }, // see schoolSchema#inMaintenance (below),
+		inUserMigration: { type: Boolean, default: false },
```

Comment:
i don't think we need the default here, we will have legacy data which has this value set to undefined.

Reply:
```suggestion
		inUserMigration: { type: Boolean },
```

Reply:
It doesn't matter in that case, because it is always checked for the value being true, but it is now removed.